packages/app-mobile/components/screens/ConfigScreen/plugins/PluginBox.tsx

```diff
@@ -106,7 +126,13 @@ const PluginBox: React.FC<Props> = props => {
 		if (!props.item.manifest._recommended || !props.isCompatible) {
 			return null;
 		}
-		return <Chip icon='crown' mode='outlined'>{_('Recommended')}</Chip>;
+		return <Chip
+			icon='crown'
+			mode='outlined'
+			onPress={() => onRecommendedPress()}
```

Comment:
You went through the trouble of creating a separate function to avoid re-rendering, yet here you create a new function on each render. So just `onPress={onRecommendedPress}`